openjdk.build/include/top.xml

```diff
@@ -191,12 +191,17 @@ limitations under the License.
 	</target>
 
 	<!-- Make sure mauve.jar is built. -->
-	<target name="run-mauve-configure" depends="fetch-prestaged-mauve-jar" unless="isAutomatedBuild">
+	
+	<target name="run-mauve-configure" unless="prestaged-prereqs-available">
 		<ant antfile="${source_root}/openjdk.test.mauve/build.xml" target="configure" dir="${source_root}/openjdk.test.mauve" inheritAll="false"/>
 	</target>
 	
-	<target name="fetch-prestaged-mauve-jar" if="isAutomatedBuild">
-		<copy file="${prereqs_root}/../TestConfig/lib/systemtest_prereqs/mauve/mauve.jar" todir="${prereqs_root}/mauve"/>
+	<target name="check-if-prestaged-prereqs-available">
+		<available file="${prereqs_root}/../TestConfig/lib/systemtest_prereqs/mauve/mauve.jar" property="prestaged-prereqs-available"/>
+	</target>
+	
+	<target name="fetch-prestaged-mauve-jar" if="prestaged-prereqs-available">
+		<copy file="${prereqs_root}/../TestConfig/lib/systemtest_prereqs/mauve/mauve.jar" todir="${prereqs_root}/mauve" failonerror="false"/>
```

Comment:
Is possible to directly get the jars into `${prereqs_root}` in the first step in Jenkins file? Feel redundant to copy it into `TestConfig/lib/systemtest_prereqs`, then move it into `${prereqs_root}/mauve`.

Reply:
The copyArtifacts step happens at the beginning of the build, at which point we still do not have the `{prereqs_root}/` folder - that gets created much later when we have cloned the systemtest repo and started the build process. This is why the two steps were needed. @llxia

Reply:
On a second try, you are right. I can directly copy into the `{prereqs_root}` directory, instead of having a second copy step. Updated the PR. Tested here: https://ci.adoptopenjdk.net/view/work%20in%20progress/job/Grinder_Sandbox/25/console @llxia